syntax = "proto3";



package user_auth;

service UserAuth {
  // Authenticates a user with email and password.
  rpc AuthenticateUser(AuthenticationRequest) returns (AuthenticationResponse) {}
}

message AuthenticationRequest {
  string email = 1;
  string password = 2;
}

message AuthenticationResponse {
  bool success = 1;
  string message = 2;
}